File: Homework 1/Homework 1/Form1.cs
namespace Homework_1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void checkBoxAccept_CheckedChanged(object sender, EventArgs e)
        {
            this.buttonSend.Enabled = this.checkBoxAccept.Checked;
        }

        private void buttonReset_Click(object sender, EventArgs e)
        {
            this.richTextBox1.Clear();
        }

        private void buttonReset_MouseHover(object sender, EventArgs e)
        {
            this.toolTip1.SetToolTip(buttonReset, "Clears the above textbox");
        }

        private void buttonSend_Click(object sender, EventArgs e)
        {
            this.richTextBox1.Enabled = false;
            this.checkBoxAccept.Enabled = false;
            this.buttonReset.Enabled = false;
            this.buttonSend.Enabled = false;
        }

        private void buttonSend_MouseHover(object sender, EventArgs e)
        {
            this.toolTip1.SetToolTip(buttonSend, "Sends the form");
        }

        private void buttonClose_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        private void buttonClose_MouseHover(object sender, EventArgs e)
        {
            this.toolTip1.SetToolTip(buttonClose, "Closes the app");
        }
    }
}

File: Homework 1_VB/Homework 1_VB/Form1.vb
Public Class Form1
    Private Sub CheckBoxAccept_CheckedChanged(sender As Object, e As EventArgs) Handles CheckBoxAccept.CheckedChanged
        Me.ButtonSend.Enabled = Me.CheckBoxAccept.Checked
    End Sub
    Private Sub ButtonReset_Click(sender As Object, e As EventArgs) Handles ButtonReset.Click
        Me.RichTextBox1.Clear()
    End Sub
    Private Sub ButtonReset_MouseHover(sender As Object, e As EventArgs) Handles ButtonReset.MouseHover
        Me.ToolTip1.SetToolTip(ButtonReset, "Clears the above textbox")
    End Sub
    Private Sub ButtonSend_Click(sender As Object, e As EventArgs) Handles ButtonSend.Click
        Me.RichTextBox1.Enabled = False
        Me.CheckBoxAccept.Enabled = False
        Me.ButtonReset.Enabled = False
        Me.ButtonSend.Enabled = False
    End Sub
    Private Sub ButtonSend_MouseHover(sender As Object, e As EventArgs) Handles ButtonSend.MouseHover
        Me.ToolTip1.SetToolTip(ButtonSend, "Sends the form")
    End Sub
    Private Sub ButtonClose_Click(sender As Object, e As EventArgs) Handles ButtonClose.Click
        Me.Close()
    End Sub
    Private Sub ButtonClose_MouseHover(sender As Object, e As EventArgs) Handles ButtonClose.MouseHover
        Me.ToolTip1.SetToolTip(ButtonClose, "Closes the app")
    End Sub
End Class
